pytorch_lightning/accelerators/accelerator.py

```diff
@@ -154,14 +154,7 @@ def root_device(self) -> torch.device:
         return self.training_type_plugin.root_device
 
     def teardown(self) -> None:
-        """
-        This method is called to teardown the training process.
-        It is the right place to release memory and free other ressources.
-
```

Comment:
i'd keep this documentation on the accelerator and training type plugin and remove it from child classes